SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -539,6 +539,9 @@ void USpatialNetDriver::OnGSMQuerySuccess()
 			WorldContext.PendingNetGame->bSuccessfullyConnected = true;
 			WorldContext.PendingNetGame->bSentJoinRequest = false;
 			WorldContext.PendingNetGame->URL = RedirectURL;
+
+			// Ensure the singleton map is reset as it will contain bad data from the old map
```

Comment:
Worth calling out that this is important for (explicitly) server travel?

Reply:
I guess this will also be remove with your changes.

Reply:
ye will rebase inabit